simulators/optimism/devnet/main.go

```diff
@@ -129,6 +129,8 @@ func runAllTests(t *hivesim.T) {
 					}
 				},
 			})
+			// TODO: debug re-org issue and remove
+			time.Sleep(5 * time.Second)
```

Comment:
This is a temporary workaround because some transactions get re-orged out. Spoke with @protolambda and decided this is OK for now.